manifests/params.pp

```diff
@@ -128,6 +128,16 @@
       $source = "${source_base}sudoers.aix"
       $config_file_group = 'system'
     }
+    darwin: {
+      $package = undef
```

Comment:
This `undef` triggers an error:

    Could not retrieve catalog from remote server: Error 400 on SERVER: Invalid relationship: File[/etc/sudoers] { require => Package[undef] }, because Package[undef] doesn't seem to be in the catalog

See other comments for the proposed fix.